BackEnd/src/main/kotlin/quest/marketstack/tradingapp/domain/closedtrades/MongoClosedTradeDataSource.kt

```diff
@@ -0,0 +1,20 @@
+package quest.marketstack.tradingapp.domain.closedtrades
```

Comment:
I renamed this package from datasource to domain. While I know the objects within it are called DataSource, in general if you have a folder where you're defining objects to be used in other places, I recommend that being called domain. I guess doesn't technically matter but at least what I've seen that's what you should do. Feel free to rename it (right click on the folder in IntelliJ, then Refactor->Rename) back if you really don't like it.